cppcheck.spec

```diff
@@ -0,0 +1,22 @@
+### RPM external cppcheck 1.89
+Source: https://github.com/danmar/cppcheck/archive/%{realversion}.tar.gz
+
+BuildRequires: cmake gmake
+Requires: pcre
```

Comment:
please add python dependency as it uses system python

https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-5583a2/2448/external/cppcheck/1.89-5583a2/log
```
-- Found PythonInterp: /usr/bin/python (found version "2.7.5") 
```